### DISCUSSION TITLE
After updating the plugins, titles won't match

### DISCUSSION THREAD

**Discussion by dimmitri9** [CodyBerenson/PGMA-Modernized#246] [Q&A] [answered]
Don't know why this is happening, but the updated plugins aren't matching any titles from now on
Server - Version 1.31.3.6868 
[com.plexapp.agents.WayBig.log](https://github.com/CodyBerenson/PGMA-Modernized/files/11560637/com.plexapp.agents.WayBig.log)
[com.plexapp.agents.Fagalicious.log](https://github.com/CodyBerenson/PGMA-Modernized/files/11560641/com.plexapp.agents.Fagalicious.log)

https://fagalicious.com/trystanbull-trystan-bull-nick-reeves-and-max-morgan-in-my-stepdad-banged-me/
https://www.waybig.com/blog/2022/11/16/gayhoopla-gio-givannis-first-time-trying-ass-play-on-camera/
https://fagalicious.com/idisruptive-films-ty-roderick-michael-boston-first-timers/

**Comment by JPH71** [comment, marked as answer]
I have just matched one 20 minutes ago...

Try again and send the log file...

Before you do this download the latest from the repository. Then save your
preferences as there has been a change... dropped the colour purple in
favour of Indigo

Also update plex media server as they have fixed the error that was
preventing the scraping...

Restart PMS and scrape a film and send me the log


Xxxx

**Reply by dimmitri9** [reply]
Thanks! =)
